Model/ResourceModel/Currency.php

```diff
@@ -36,8 +36,39 @@ public function getByCurrency($code)
         return $connection->fetchAll($select);
     }
 
+    //should write  deletion routine too
+
+    /**
+     * @param $currencyCode string
+     * @return array
+     */
+    public function getPrecisionByCurrency($currencyCode)
+    {
+        $connection = $this->getConnection();
+        $select = $connection->select()
+            ->from($this->getTable('reach_currency_precision'))
+            ->where('currency_code = ?', $currencyCode);
+        return $connection->fetchAssoc($select);
+    }
+
+    /**
+     * @param $currencyCode string
+     * @param $precision integer
+     * @return \Zend_Db_Statement_Interface
+     */
+    public function setPrecisionByCurrency($currencyCode, $precision)
```

Comment:
To save precision info into the database. I think this comment should reside in the code itself.